portfolio/src/main/webapp/script.js

```diff
@@ -12,15 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function getPhrase(){
-    fetch('data').then((response) => response.json()).then((list) => {
-        document.getElementById('dynamic-description').innerText = list;
-    });
+async function getPhrase(){
+    let response = await fetch('/load-data');
+    let list = await response.json();
+    if(list.length != 0){
```

Comment:
You probably don't need this if statement.

The for loop won't run if list.length = 0

Reply:
That's a good point. I think I forgot to remove that from a previous commit, I'll fix it and send the edit